models/forms/ContainerSettings.php

```diff
@@ -62,18 +71,40 @@ public function rules()
     {
         return [
             [['snippetGallery', 'hideSnippet'], 'integer'],
-            [['snippetGallery'], 'containerGallery']
+            [['snippetGallery'], 'containerGallery'],
+            [['sortOrder'], 'validateSortOrder'], //'in', 'range' => [0,100], 'message'=>'Please enter a value from 0 to 300 in increments of 100'
```

Comment:
Why not use Yii2 build in number validator?
https://www.yiiframework.com/doc/api/2.0/yii-validators-numbervalidator

```
[['sortOrder'], 'number', 'min' => static::SORT_MIN, 'max' => static::SORT_MAX]
```

Reply:
Good day,@luke-!

I have created a custom validation to have more advanced validation and output messages.
It checks whether the value is in the range from MIN and MAX and with a step of 100 only. So the values that are only acceptable is this kind of list [0, 100, 200...5000] 

if you want to have no restrictions as you described, we can make it as you have mentioned above.

Thank you!